src/RatingSymbol.jsx

```diff
@@ -35,13 +35,14 @@ class RatingSymbol extends React.PureComponent {
       display: 'inline-block',
       position: 'absolute',
       overflow: 'hidden',
+      lineHeight: 0,
```

Comment:
Is this really necessary?

Reply:
After some thought `lineHeigth: unset` might also work and definitely will be cleaner solution.
I'll check that and will get back to you in the evening.
It was necessary in my case: component I've pasted above can be used as `inline-block` next to some text that has high `line-height` and if span in `react-rating` inherits that `line-height`, then border around `react-rating` component becomes taller as well, as it has to stretch to accommodate for `react-rating`'s height.
I'll post some screen in the evening along with results of my testing of `unset` param.